float32 position  # Position value between 0.0 and 1.0
uint8 servo_id    # ID of the servo to control
bool activate     # True to activate servo movement